### DISCUSSION TITLE
Quick Option to Change Annotation/Highlight Color

### DISCUSSION THREAD

**Discussion by abelokoj-github** [sumatrapdfreader/sumatrapdf#3049] [Ideas]
It will be good if we have something like this from Xodo pdf to easily access the option to access highlighting color in Sumatra pdf. We can have a single right/left click to bring up the small window like in the first figure that gives option to change annotation/highlight color without going into the full annotation settings. There can be other key-bindings like say **CTRL+CLICK** to access the advanced annotation/highlight editing options.  The _**Second picture below pops up when you click the color palette icon  in the fist figure**_

The user should also be able to select which colors will show up (i.e favorite colors) when the color palette in the second figure shows up. Then more advanced annotation/highlighting editing can be done when you click **advanced**

I think it is way more intuitive to see the original colors than having the name of the colors like Sumatra currently has. 

Thanks for your kind considerations.

![image](https://user-images.githubusercontent.com/69094732/197402934-672abc79-b040-4db1-80fd-8a4058e00080.png)

**Second picture below pops up when you click the color palette icon  in the fist figure**

![image](https://user-images.githubusercontent.com/69094732/197402938-76242113-f704-4c84-9321-7c2bf4d0d444.png)

**Comment by elleffangegjelstad** [comment]
I agree; will, however, try AutoHotkey as a temporary solution.

**Comment by elleffangegjelstad** [comment]
I tend to use colours "semantically"; e.g. Example→Green, Definitions→Cyan, Structure→Orange, Disagree→Purple etc, so it would be great to define my own colours with my own names.
